osquery/tables/applications/darwin/browser_plugins.cpp

```diff
@@ -103,7 +126,20 @@ QueryData genBrowserPlugins(QueryContext& context) {
       auto dir = fs::path(row.at("directory")) / kBrowserPluginsPath;
       if (listDirectoriesInDirectory(dir, bundles).ok()) {
         for (const auto& dir : bundles) {
-          genBrowserPlugin(row.at("uid"), dir, results);
+          // Check if the 'plugin' we're looking at is the Disabled folder
+          // if it is we enumerate the plugins under that dir, and set a
+          // disabled flag.
+          if(dir.find("Disabled") != std::string::npos){
```

Comment:
I suggest changing this up a bit and adding a secondary loop outside of this that enumerates the disabled directory.